src/de.rs

```diff
@@ -104,23 +104,32 @@ pub fn from_str<'de, T>(s: &'de str) -> Result<T, Error>
 ///     name: String,
 /// }
 ///
-/// fn main() {
-/// # }
-/// # fn fake_main() {
-///     let mut file = File::open("config.toml").unwrap();
-///     let config: Config = toml::from_reader(&mut file).unwrap();
+/// fn main() -> Result<(), io::Error> {
+///     let toml_str = r#"
+///         title = 'TOML Example'
+///
+///         [owner]
+///         name = 'Lisa'
+///     "#;
+///     let mut reader = io::Cursor::new(toml_str);
```

Comment:
```suggestion
///     // to read file instead
///     // let mut file = File::open("config.toml").unwrap();
```

Maybe we should add that here, I don't know what is `io::Cursor` before reading this.

Reply:
I know only one Cursor in rust and it's io::Cursor that implement Read/Write interface like std::fs::File. I put this example in order to not link the tested code with the structure of the project. Can you share to me the Cursor that you know ?

Reply:
Err, I don't know any Cursor. But I meant use `Cursor` as in `use std::io::Cursor`.